Signal/src/network/http/RPServerRequestsManager.m

```diff
@@ -36,7 +36,7 @@ - (id)init{
         NSURL *endPointURL = [NSURL URLWithString:[NSString stringWithFormat:@"https://%@:%hu", endpoint.hostname, endpoint.port]];
         NSURLSessionConfiguration *sessionConf = NSURLSessionConfiguration.defaultSessionConfiguration;
         self.operationManager = [[AFHTTPSessionManager alloc] initWithBaseURL:endPointURL sessionConfiguration:sessionConf];
-        self.operationManager.securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModeNone];
+        self.operationManager.securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModePublicKey];
         self.operationManager.securityPolicy.allowInvalidCertificates = YES;
```

Comment:
OK! rest of this makes sense. maybe wait and see if Square merges in if that makes sense too.